### PR TITLE
Organize pom's

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

**Comment by danielpassos** [comment]
@secondsun :eyeglasses: ?

File: aerogear-android-auth-test/pom.xml [hunk 1]
```diff
@@ -30,9 +30,19 @@
...
-        <dexmaker.mockito.version>1.2</dexmaker.mockito.version>
+        <dexmaker.version>1.0</dexmaker.version>
+        <dexmaker.mockito.version>1.0</dexmaker.mockito.version>
+        <android.support.test.version>0.3</android.support.test.version>
+
+        <!-- Plugins parameters -->
+        <android.maven.plugin.dex.xms>-Xms256m</android.maven.plugin.dex.xms>
+        <android.maven.plugin.dex.xmx>-Xmx2048m</android.maven.plugin.dex.xmx>
+
+        <maven.javadoc.skip>true</maven.javadoc.skip>
```
Comment (secondsun) [inline aerogear-android-auth-test/pom.xml]: Is this property used?

**Comment by secondsun** [comment]
@danielpassos Other than my question about the javadoc property, +1.

File: aerogear-android-auth-test/pom.xml (hunk 1 above)
Comment (danielpassos) [inline aerogear-android-auth-test/pom.xml]: Not in this library/module. Removed :)

**Comment by secondsun** [comment]
landed

**Pull request merged**